File: Remove Duplicates from Sorted List/Remove Duplicates from Sorted List.php
<?php

//  Definition for a singly-linked list.
class ListNode {
    public $val = 0;
    public $next = null;
    function __construct($val = 0, $next = null) {
        $this->val = $val;
        $this->next = $next;
    }
}

class Solution {

    /**
     * @param ListNode $head
     * @return ListNode
     */
    function deleteDuplicates($head) {
        $items = [];
        $new_head = $prev_item = new ListNode();

        while($head != null){
            if(count($items) == 0 || !in_array($head->val, $items)){
                array_push($items, $head->val);
                $prev_item->next = new ListNode($head->val);
                $prev_item = $prev_item->next;
            }
            $head = $head->next;
        }
        return $new_head->next;
    }
}

?>

File: Remove Duplicates from Sorted List/Remove Duplicates from Sorted List.py
# Definition for singly-linked list.
class ListNode(object):
    def __init__(self, val=0, next=None):
        self.val = val
        self.next = next
        
class Solution(object):
    def deleteDuplicates(self, head):
        """
        :type head: ListNode
        :rtype: ListNode
        """

        items = set()
        new_head = prev_item = ListNode()
        while head:
            if head.val not in items:
                items.add(head.val)
                prev_item.next = ListNode(head.val)
                prev_item = prev_item.next
                
            head = head.next
            
        return new_head.next